website/www/site/content/en/get-started/quickstart-py.md

```diff
@@ -15,7 +15,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Apache Beam Python SDK Quickstart
+# WordCount quickstart for Python
```

Comment:
The original plan was to deprecate the WordCount quickstarts later. I'm open either way. I think if we remove the old one, it should be a separate PR, as their are a lot of links to it in the docs.

Reply:
SG to deprecate it later. It would be good to unify shared content. I wonder if we should reference new guide in the old guide? Specific portions we could update:

`## Set up your environment` in new guide could refer to `## Set up your development environment` in the old guide.

Reply:
`The Python SDK supports Python 3.6, 3.7, and 3.8. Beam 2.24.0 was the last release with support for Python 2.7 and 3.5.` can be removed as it is covered in the new guide under `### Check your Python version`.

cc: @AnandInguva FYI.

Reply:
fixed